CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 
 FEATURES:
+* r/tfe_workspace: Add attribute `resource_count` to `tfe_workspace` ([#682](https://github.com/hashicorp/terraform-provider-tfe/pull/682))
```

Comment:
About your comment `interestingly the TFE provider changelog doesn't indicate who added PRs - do we want to start doing that?`, yes, I think it would be nice to add the name of the external contributor, if this doesn't get added automatically 😊

Reply:
I agree with you! I'll add that.